pkg/config/config.go

```diff
@@ -207,6 +208,11 @@ type Config struct {
 	// Public: No
 	MemProfile string `yaml:"mem_profile" envconfig:"mem_profile" public:"false"`
 
+	// Interval in seconds to create memory profile snapshots, default 300 seconds (5min)
+	// Default: 300
+	// Public: No
```

Comment:
I guess we aren't updating docs with this one

Reply:
this is not a public one, so I guess no